src/lib/local-storage.js

```diff
@@ -56,7 +43,7 @@ class LocalStorage implements IStorage {
     }
 
     storage.createPackage(name, generatePackageTemplate(name), err => {
-      if (_.isNull(err) === false && err.code === fileExist) {
+      if (_.isNull(err) === false && err.code === STORAGE.FILE_EXIST) {
```

Comment:
`STORAGE.FILE_EXIST` I'd rather call the namespace something as `XXX_ERROR`. The idea is should be used by plugins and some devs might find it misleading.

Reply:
The same for `STORAGE.NO_SUCH_FILE`